plugins/poll/config/locales/client.en.yml

```diff
@@ -22,7 +22,9 @@ en:
           title: "Results will be shown once <strong>closed</strong>."
         staff:
           title: "Results are only shown to <strong>staff</strong> members."
-
+        tabs:
+          votes: "Votes"
+          outcome: "IRV Outcome"
```

Comment:
Is "IRV" needed in this text? I'd avoid using acronyms.

Reply:
@gschlager I would recommend keeping things short because this specific instance is a tabbed interface:

![image](https://github.com/discourse/discourse/assets/35533304/92f33c6c-3db5-4307-a659-a89bc3b1de47)

I did experiment with "Instant Run-off Voting Outcome" but it looked a bit strange having one tab button much longer than the other.

thoughts?

Reply:
I think it's sufficient to use "Outcome"

Reply:
done, thanks!